src/map/utils/petutils.h

```diff
@@ -29,6 +29,7 @@
 #include "../../common/mmo.h"
 
 #include "../entities/petentity.h"
+#include "../entities/trustentity.h"
```

Comment:
i think we can just forward declare CPetEntity and CTrustEntity (like we already have done with CBattleEntity in this file)

Reply:
Done.